src/styles/main.scss

```diff
@@ -0,0 +1,67 @@
+body {
+  background: rgb(86, 71, 224);
+  margin: 0;
+}
+
+.clock {
+  margin: 100px auto;
```

Comment:
<img width="1002" alt="image" src="https://user-images.githubusercontent.com/102411130/206658303-a926156a-cd73-4b16-92af-6139222ac16f.png">
check examples for this point in a checklist